src/styles/tools/_mixins.scss

```diff
@@ -8,3 +8,15 @@ $breakpoint-tablet: 820px;
     background-size: 30%;
   }
 }
+
+@mixin doodle($height, $width, $image_url, $background_size, $rotate) {
+  content: "";
+  position: absolute;
+  height: $height;
+  width: $width;
+  background-image: url($image_url);
+  background-size: $background_size;
+  background-repeat: no-repeat;
+  rotate: $rotate;
+  z-index: 2;
```

Comment:
This mixin's pretty complicated, I think what I was going for was something a little simpler to save some typing. Having 5 parameters without default values is pretty difficult to use without looking up what they are.

I'd maybe set up the mixin like this:
```scss
@mixin doodle {
  content: "";
  position: absolute;
  background-repeat: no-repeat;
  z-index: 2;
}
```

And then usage might be like:
```scss
::after {
  @include doodle;

  height: 20%;
  width: 70%;
  background-image: url("../../../public/images/doodles/doodle_line.avif");
  background-size: cover;
  rotate: -30deg;
}